bundles/core/src/main/java/com/adobe/cq/forms/core/components/models/form/Base.java

```diff
@@ -205,6 +207,29 @@ default String getDescription() {
         return null;
     }
 
+    /**
+     * Returns the short description of the field
+     *
+     * @return the short description of the field
+     * @since com.adobe.cq.forms.core.components.models.form 1.0.0
+     */
+    @JsonIgnore
+    @Nullable
+    default String getShortDescription() {
```

Comment:
Can you move this to Field since this `shortDescription` and `isShortDescriptionVisible` is part of form field and not form panel